<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2048</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="Label_Hero_Hp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hero_Hp:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Kill_Count">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>0</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>击杀数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Hero_Mp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hero_Mp:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Kill_Goal">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>90</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目标击杀：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Help">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
